pkg/metrics/common.go

```diff
@@ -23,6 +23,8 @@ import (
 )
 
 var (
+	// StartingTime represents
+	// the starting of time
```

Comment:
@rbo13 We can have this comment in one line only rather than having it in two lines since its not a very big comment !

Reply:
And also the comment should be more specific like whose starting time it represents !